moped-editor/src/components/GridTable/Filters.js

```diff
@@ -605,18 +625,25 @@ const GridTableFilters = ({
               <Hidden mdDown>
                 <Grid item xs={12} md={1} style={{ textAlign: "center" }}>
                   <IconButton
-                    disabled={Object.keys(filterParameters).length === 1 && !filterComplete}
+                    disabled={
+                      Object.keys(filterParameters).length === 1 &&
+                      !filterComplete
```

Comment:
i don't know if this is helpful or necessary, but fwiw at a glance it looks like the `filterComplete` value makes more sense as something returned from `useMemo` (or not even memoized 😈 ) vs a useEffect + state var.